src/matcha_ml/infrastructure/resources/output.tf

```diff
@@ -14,6 +14,7 @@ output "pipeline_zenml_storage_path" {
   value       = module.zenml_storage.zenml_blobstorage_container_path
 }
 
+
```

Comment:
I assume we will add a storage connection string output in a PR after this? Similar to:
```
output "data_versioning_dvc_azure_connection_string" {
  description = "The Azure connection string for the DVC data storage"
  value       = module.dvc.primary_connection_string
  sensitive   = true
}
```

Reply:
This can be ignored, I have read the note in the ticket.